src/inmanta/moduletool.py

```diff
@@ -1328,9 +1326,9 @@ def release(
             self.release(dev=True, message="Bump version to next development version", patch=True)
 
 
-class ModuleChangelog:
```

Comment:
For backward compatibility I would add `ModuleChangelog = Changelog` below this class